### PR TITLE
Stop using Hydra-head

### PR DESCRIPTION
**Pull request by jcoyne**
So that we can upgrade to Rails 5

### CODE REVIEW

File: app/models/hydra/datastream/rights_metadata.rb [hunk 1]
```diff
@@ -0,0 +1,270 @@
+require 'active_support/core_ext/string'
```
Comment (mjgiarlo) [inline app/models/hydra/datastream/rights_metadata.rb]: Why is this file needed? Is it because it was defined in hydra-head, and hydra-access-controls depends on it?

Comment (mjgiarlo) [reply app/models/hydra/datastream/rights_metadata.rb]: @jcoyne 	Worth a shot. Have a preference for doing that in this branch or in follow-up work?

Comment (jcoyne) [reply app/models/hydra/datastream/rights_metadata.rb]: I'd like to do that elsewhere.  I prefer to keep PRs small as possible.

Comment (mjgiarlo) [reply app/models/hydra/datastream/rights_metadata.rb]: You bet.
